docs/source/reference/volumes.rst

```diff
@@ -0,0 +1,47 @@
+Volumes
+-------
+
+This section lists the different persistent volume claims that are created during a standard deployment of Substra in a cluster.
+
+Orchestrator
+************
+
+The orchestrator claims are only linked with its database. This database (and the underlying volume) is important has it is where we store all the events that happen in the network.
+
++--------------+--------------+----------+---------------------------------+--------------------------+---------------+----------------+---------------+----------------------------------------------+---------+
+|   PVC name   |  Component   |   Pod    |           access mode           | Volume Size default (Gi) | storage class | reclaim policy | Can be reused |                How to re use                 | Comment |
++==============+==============+==========+=================================+==========================+===============+================+===============+==============================================+=========+
+| Postgres PVC | orchestrator | postgres | ReadWriteOnce (can be modified) | 8                        | <empty>       | default        | yes           | postgresql.primary.persistence.existingClaim |         |
++--------------+--------------+----------+---------------------------------+--------------------------+---------------+----------------+---------------+----------------------------------------------+---------+
+
+
+Backend
+*******
+
+The backend is the more compolex component and requires different volumes for functioning. The ones that requires to be kept are re-usable and can be created outside of the deployment of the Substra stack. The other ones act as cache, and there is no way for now to re-use existing volumes using the Helm chart.
+
++---------------------+-----------+------------+---------------------------------+--------------------------+---------------+----------------+---------------+----------------------------------------------+----------------------------------------------------+
+|      PVC name       | Component |    Pod     |           access mode           | Volume Size default (Gi) | storage class | reclaim policy | Can be reused |                How to re use                 |                      Comment                       |
```

Comment:
Capitalisation of the PVC name column does not feel consistent, should we use the kube name here? if that's already the case, shall it be put in monospace font?

Reply:
I will harmonize the capitalization. I cannot put the name of the PVC, as some of them can use already existing volumes, and the other ones are generated using current namespace, deployment name etc...